packages/hardhat/contracts/MyUSDEngine.sol

```diff
@@ -24,19 +26,19 @@ contract MyUSDEngine is Ownable {
     address private i_rateController;
 
     uint256 public borrowRate; // Annual interest rate for borrowers in basis points (1% = 100)
-    uint256 public lastUpdateTime;
 
     // Total debt shares in the pool
     uint256 public totalDebtShares;
 
     // Exchange rate between debt shares and MyUSD (1e18 precision)
     uint256 public debtExchangeRate;
+    uint256 public lastUpdateTime;
 
     mapping(address => uint256) public s_userCollateral;
     mapping(address => uint256) public s_userDebtShares;
 
     event CollateralAdded(address indexed user, uint256 indexed amount, uint256 price);
-    event CollateralWithdrawn(address indexed from, address indexed to, uint256 indexed amount, uint256 price);
+    event CollateralWithdrawn(address indexed withdrawer, uint256 indexed amount, uint256 price);
```

Comment:
nice, `from` and `to` will always be the same right?

Reply:
Yes. There was a time when this same event covered both withdrawals and liquidations but we added a Liquidation event but we forgot to change this to only have the withdrawer.